static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -270,7 +270,8 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
       ) : (
         newDocs?.platformOptions &&
         widgetPlatform &&
-        !crashReportOnboarding && (
+        !crashReportOnboarding &&
+        !isLoading && (
```

Comment:
added this in bc some text was loading in before the platform select was ready and it looked weird